View/script/script.js

```diff
@@ -0,0 +1,15 @@
+// Function to toggle between dark and light mode.
+let dark=false;
```

Comment:
Why are we setting dark as false here ?
If this is set to false 
```
    $('.navbar').addClass('dark-mode-navbar');
    $('body').addClass('dark-mode-body');
```

This two classes are going to get executed everytime.

We need to store the user's preferred choice of light or dark mode, and then based on the choice update UI accordingly. Is that requirement being met here?